src/types/fee_multiple.ts

```diff
@@ -0,0 +1,42 @@
+import BigNumber from 'bignumber.js';
+
+export class FeeMultiple {
+	constructor(private readonly feeMultiple: number) {
+		if (this.feeMultiple < 1.0 || Number.isNaN(feeMultiple) || !Number.isFinite(feeMultiple)) {
+			throw new Error('Fee multiple must be >= 1.0!');
+		}
+	}
+
+	[Symbol.toPrimitive](hint?: string): string | number {
+		if (hint === 'string') {
+			return this.toString();
+		}
+
+		return this.feeMultiple;
+	}
+
+	toString(): string {
+		return `${this.feeMultiple}`;
+	}
+
+	valueOf(): number {
+		return this.feeMultiple;
+	}
+
+	equals(other: FeeMultiple): boolean {
+		// Hard to say what precision to use here. Use EPSILON by default since most multiples will be pretty small
```

Comment:
We actually don't use it in the codebase at all. Also, `FeeMultiple` doesn't implement `Equatable<T>`
Maybe we don't need this method in this specific type (?

Reply:
I'm inclined to agree. I'll remove it.

Reply:
54285dd